### DISCUSSION TITLE
2022/12/29/%E5%A6%82%E4%BD%95%E5%BD%A2%E6%88%90%E8%87%AA%E5%B7%B1%E7%9A%84%E8%A7%82%E5%BD%B1%E5%93%81%E5%91%B3/

### DISCUSSION THREAD

**Discussion by giscus** [WyattJia/WyattJia#79] [Announcements]
# 2022/12/29/%E5%A6%82%E4%BD%95%E5%BD%A2%E6%88%90%E8%87%AA%E5%B7%B1%E7%9A%84%E8%A7%82%E5%BD%B1%E5%93%81%E5%91%B3/

https://eventloop.live/2022/12/29/%E5%A6%82%E4%BD%95%E5%BD%A2%E6%88%90%E8%87%AA%E5%B7%B1%E7%9A%84%E8%A7%82%E5%BD%B1%E5%93%81%E5%91%B3/

**Comment by WyattJia** [comment]
first commit

**Reply by WyattJia** [reply]
😄
